Models/index.js

```diff
@@ -1,8 +1,12 @@
-const {selectTopics} = require('./selectTopics')
-const {selectEndPoints} = require('./selectEndPoints')
-const {selectArticles, selectArticlesById} = require('./selectArticles')
-const {selectCommentsForArticle} = require('./selectComments')
-const {insertComment} = require('./selectComments')
+const {selectTopics} = require('./topicsModels')
+const {selectEndPoints} = require('./endPointsModels')
+const {selectArticles, selectArticlesById} = require('./articlesModels')
+const {selectCommentsForArticle} = require('./commentsModels')
+const {insertComment} = require('./commentsModels')
+const {checkArticleExists} = require('./articlesModels')
+const {updateArticle} = require('./articlesModels')
+const {removeComment} = require('./commentsModels')
+const {checkCommentExists} = require('./commentsModels')
```

Comment:
To make this file a lot cleaner, consider combing required in functions that come from the same file